state/state.go

```diff
@@ -15,8 +18,11 @@ type Tx = coretypes.Tx
 // TxResponse is an alias to the TxResponse type from Cosmos-SDK.
 type TxResponse = sdk.TxResponse
 
-// Address is an alias to the Address type from Cosmos-SDK.
-type Address = sdk.Address
+// Address is an alias to the Address type from Cosmos-SDK. It is embedded into a struct to provide
```

Comment:
Its technically not an alias anymore

Reply:
Also, why not 'type Address sdk.Address'? Without =

Reply:
Ah, because address is an interface?

Reply:
Reflection cannot work on interface types like that, because you never know which concrete type to marshal a value back into. I need to double check that no other api endpoints are returning interfaces - because it breaks the server